Cython/Compiler/ExprNodes.py

```diff
@@ -774,6 +776,10 @@ def calculate_result_code(self):
 #        #  Release temporaries used by LHS of an assignment.
 #        self.release_subexpr_temps(env)
 
+    def call_funcstate_allocate_temp(self, code, type):
+        # implemented so it can be overidden in special cases
+        return code.funcstate.allocate_temp(type, manage_ref=self.use_managed_ref)
```

Comment:
I'd just call this method `allocate_temp`. I don't really see a reason (nor a good way) to distinguish it from the method of code writers. In any case, given the more complex implementation further down, the name is really misleading because it's very specific about what it does but does more than that.